libs/eo/core/shell/src/lib/eo-core-shell.providers.ts

```diff
@@ -25,6 +26,7 @@ import { danishDatetimeProviders } from '@energinet-datahub/watt/danish-date-tim
 import { WattModalService } from '@energinet-datahub/watt/modal';
 
 export const eoCoreShellProviders = [
+  FormGroupDirective,
```

Comment:
Question: This doesn't seem to be necessary. Do you have a specific reason for adding this? Otherwise, please remove it :)

Reply:
I needed to add this for reactive form validation in watt-field to work
https://github.com/Energinet-DataHub/greenforce-frontend/blob/ae88fffd8c922243373df5b7a7bb742ec0bcd907/libs/watt/src/lib/components/field/watt-field.component.ts#L63

Reply:
But if I remove it, it still work? (in EO)

Reply:
Aha, must be because you added the optional param